config/config.go

```diff
@@ -1,7 +1,8 @@
-package main
+package config
 
 import (
 	"encoding/json"
+	sideline_models "github.com/flipkart-incubator/go-dmux/sideline-models"
```

Comment:
what is the intent of keeping models separate from sideline package itself?

Reply:
in dmux.go of core only sideline-models are required.
sideline package has only the code which will imported and called from a main function of the client. 
since both the functionalities are different thought of having them in different modules.